src/app/issue/dashboard/dashboard.component.html

```diff
@@ -70,6 +71,13 @@
   <tbody>
     <tr (click)="routeToIssue(data.id)" *ngFor="let data of filterTable.data">
       <td>{{ data.issueDetail.name }}</td>
+      <td>
+        <app-timer-button
+          [disabled]="cannotStartTime(data.state?.phase)"
```

Comment:
Das Disabled des Buttons würde ich noch direkt in die TimerButton Komponente übernehmen